src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,142 @@
+package duke;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * this class contains the task list
+ * it will handle all task commands
+ * @author Kaiyi
+ */
+public class TaskList {
+
+    ArrayList<Task> listOfInputs = new ArrayList<>();
+    String directory;
+    String filePath;
+
+    public TaskList (String directory, String filePath) {
+        this.directory = directory;
+        this.filePath = filePath;
+
+    }
+    Storage storage = new Storage(directory,filePath);
+
+    /**
+     * print out the lists of tasks
+     */
+    public void list() {
+        System.out.println("Here are your task(s):");
+        for (int i = 0; i < listOfInputs.size(); i++) {
+            System.out.println(((i + 1) + ". " + listOfInputs.get(i).message()));
+        }
+    }
+
+
+    public void find(String word) {
+        boolean hasWord = false;
+        for (int i = 0; i < listOfInputs.size(); i++) {
+            if (listOfInputs.get(i).description.contains(word)) {
+                System.out.println(listOfInputs.get(i).message());
+                hasWord = true;
+            }
+            if (i == listOfInputs.size() - 1 && !hasWord) {
+                System.err.println("No task with the word: " + word);
+                break;
+            }
+        }
+    }
+
+    public void mark( int curr_no) throws IOException {
```

Comment:
`curr_no` should be in camelCase as well according to coding standard, such as `currNo` or `taskIndex`. Noted the same issue in other places as well.